sdk/storage/azure-storage-file-datalake/src/main/java/com/azure/storage/file/datalake/DataLakePathClientBuilder.java

```diff
@@ -601,4 +610,24 @@ public DataLakePathClientBuilder serviceVersion(DataLakeServiceVersion version)
         return this;
     }
 
+    /**
+     * Sets the {@link CustomerProvidedKey customer provided key} that is used to encrypt blob contents on the server.
```

Comment:
```suggestion
     * Sets the {@link CustomerProvidedKey customer provided key} that is used to encrypt path contents on the server.
```

Not sure if this should be path or file